docs/workflow-controller-configmap.yaml

```diff
@@ -41,6 +41,13 @@ data:
   nodeEvents: |
     enabled: true
 
+  # Whether or not to emit events on workflow completion. These can take a up a lot of space in
```

Comment:
It seems to not be on completion per the code you modified, but actually on any status change, if I'm not mistaken.

Also we can shorten / replace the following sentence by saying "See nodeEvents above"